src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -5320,20 +5471,36 @@ def _make_onedeploy_request(params):
         response = send_raw_request(params.cmd.cli_ctx, "PUT", deploy_url, body=body)
         poll_async_deployment_for_debugging = False
 
-    # check the status of async deployment
+    # check the status of deployment
     if response.status_code == 202 or response.status_code == 200:
         response_body = None
         if poll_async_deployment_for_debugging:
-            logger.info('Polling the status of async deployment')
-            response_body = _check_zip_deployment_status(params.cmd, params.resource_group_name, params.webapp_name,
-                                                         deployment_status_url, params.slot, params.timeout)
-            logger.info('Async deployment complete. Server response: %s', response_body)
+            logger.warning('Polling the status of %s deployment. Start Time: %s UTC',
+                           "async" if params.is_async_deployment else "sync",
+                           datetime.datetime.now(datetime.timezone.utc))
+            if params.track_runtime_status is not None and params.track_runtime_status:
+                # once deploymentstatus/latest is available, we can use it to track the deployment
+                deployment_id = _get_latest_deployment_id(params.cmd, params.resource_group_name,
+                                                          params.webapp_name, deployment_status_url, params.slot)
+                if deployment_id is None:
+                    raise CLIError("Unable to fetch deployment id for this deployment. "
```

Comment:
instead of raising error, let's add a log saying couldn't track runtime status and fallback to kudu endpoint